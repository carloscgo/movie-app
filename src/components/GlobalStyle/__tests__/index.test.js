import React from 'react'
import { render } from '@testing-library/react'

import GlobalStyle from '..'

const getRenderedComponent = ({ ...args }) => render(<GlobalStyle {...args} />)

describe('<GlobalStyle/>', () => {
  it('Must render the component', () => {
    const container = getRenderedComponent({})

    expect(container.baseElement).toMatchSnapshot()
  })
})
