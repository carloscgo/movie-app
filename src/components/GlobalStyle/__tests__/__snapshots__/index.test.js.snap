// Jest Snapshot v1, https://goo.gl/fbAQLP

exports['<GlobalStyle/> Must render the component 1'] = `
<body>
  <div />
</body>
`
